components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/analytics/APIMgtUsageHandler.java

```diff
@@ -45,6 +47,14 @@
     public boolean handleRequest(MessageContext mc) {
 
         TracingSpan span = null;
+
+        String apiType = mc.getProperty(APIConstants.API_TYPE).toString();
+        if (APIConstants.GRAPHQL_API.equals(apiType)) {
+            ((Axis2MessageContext) mc).getAxis2MessageContext().
+                    setProperty(Constants.Configuration.HTTP_METHOD, mc.getProperty(APIConstants.HTTP_VERB));
+            mc.setProperty(APIConstants.API_ELECTED_RESOURCE, "/*");
```

Comment:
better to make "/*" constant

Reply:
Fixed in 413eb4748a5e0a6ad71d4e9a19db6da8c62c6832